ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs

```diff
@@ -35,7 +35,7 @@
 using SBSDK = DocumentSDK.MAUI.Native.Droid.ScanbotSDK;
 using IO.Scanbot.Mrzscanner.Model;
 
-namespace DocumentSDK.NET.Droid
+namespace ReadyToUseUI.Droid
 {
     [Activity(Label = "Ready-to-use UI Demo", MainLauncher = true, Icon = "@mipmap/icon")]
```

Comment:
@scan-mayank, since we have multiple projects now, I think it makes sense to also align the labels of each app. 

For the apps I have tested on Android, I have seen:
* Ready-to-use UI Demo
* MAUI RTU UI
* Scanbot SDK Example Xamarin

Perhaps it makes sense to name them in the same way that they are structured, thus maybe "Ready-to-use UI Demo" becomes "ScanbotSDK.NET Ready-to-use UI Demo" or something of that nature, and then adjusting the rest of the apps to be consistent with each other.